src/themes/dojo/variables.css

```diff
@@ -55,4 +49,23 @@
 	--calendar-disabled-bg: var(--dojo-light-gray);
 	--calendar-hover-bg: var(--dojo-light-blue);
 	--calendar-selected: var(--dojo-blue);
+
+	/* New theme variables */
+	--dojo-white: #FFFFFF;
+	--dojo-gray: #D6DDE2;
+	--dojo-light-gray: #F4F6F7;
+	--dojo-dark-gray: #5C6C7C;
+
+	--pane-bg: var(--dojo-white);
```

Comment:
Agreed. I use `--pane-bg` in a few components so far, so I'll leave this in for now.